src/benchq/algorithms/ld_gsee.py

```diff
@@ -0,0 +1,100 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+"""Tools for the Fourier-Filtered Low-Depth Ground State Energy Estimation (FF-LD-GSEE)
+algorithm (arXiv:2209.06811v2)."""
+
+import numpy as np
+
+
+def _get_sigma(
+    alpha: float, energy_gap: float, square_overlap: float, precision: float
+) -> float:
+    """Get the standard deviation of the Gaussian convolution function (Eq. 15 of
+    arXiv:2209.06811v2).
+
+    See get_ldgsee_num_iterations for argument descriptions.
+
+    Returns: The standard deviation of the Gaussian convolution function."""
+    delta = precision**alpha * energy_gap ** (1 - alpha)
+    return min(
+        0.9
+        * delta
+        / (np.sqrt(2 * np.log(9 * delta * precision ** (-1) * square_overlap ** (-1)))),
+        0.2 * delta,
+    )
+
+
+def _get_epsilon_1(precision: float, square_overlap: float, sigma: float) -> float:
+    """Get the error with respect to the convolution function (Eq. 28 of
+        arXiv:2209.06811v2).
+
+    See get_ldgsee_num_iterations for argument descriptions.
+
+    Returns: The error with respect to the convolution function."""
+
+    return 0.1 * precision * square_overlap / (np.sqrt(2 * np.pi) * sigma**3)
+
+
+def get_ld_gsee_max_evolution_time(
```

Comment:
Change get_ld_gsee_max_evolution_time to get_ff_ld_gsee_max_evolution_time since we may want to also include a get_rs_ld_gsee_max_evolution_time function for the rejection sampling method.

Reply:
Fixed in e8f50ff9157b7bd8240c7994654d7c047b1d9041